Bottom: 0d1b5c06c711dbfa83a645fc84b1efa67b9755ab
Top:    7659bba935b8462a10edb8bb2f0960c1f0a99b81
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-07 01:30:10 +0000

Refresh of revert-always-initialise

---

diff --git a/src/create.c b/src/create.c
index 2f7bc35..7da96ee 100644
--- a/src/create.c
+++ b/src/create.c
@@ -229,13 +229,12 @@ CreateNewPCBPost (PCBType *pcb, int use_defaults)
     {
       if (ParseGroupString (Settings.Groups, &pcb->LayerGroups, &pcb->Data->LayerN))
 	return 1;
-    }
-
-  pcb->Data->Layer[top_silk_layer].Name = strdup ("top silk");
-  pcb->Data->Layer[bottom_silk_layer].Name = strdup ("bottom silk");
-  pcb->Data->Layer[top_soldermask_layer].Name = strdup ("top soldermask");
-  pcb->Data->Layer[bottom_soldermask_layer].Name = strdup ("bottom soldermask");
 
+      pcb->Data->Layer[top_silk_layer].Name = strdup ("top silk");
+      pcb->Data->Layer[bottom_silk_layer].Name = strdup ("bottom silk");
+      pcb->Data->Layer[top_soldermask_layer].Name = strdup ("top soldermask");
+      pcb->Data->Layer[bottom_soldermask_layer].Name = strdup ("bottom soldermask");
+    }
   return 0;
 }
